src/main/java/de/tum/in/www1/artemis/service/ModelingSubmissionService.java

```diff
@@ -4,14 +4,18 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
-import org.slf4j.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.server.ResponseStatusException;
 
-import de.tum.in.www1.artemis.domain.*;
-import de.tum.in.www1.artemis.domain.enumeration.*;
```

Comment:
maybe we should change it back to `.domain.*`

Reply:
or at least we should keep it consistent for the `XSubmissionsService.java` files